cpp/src/strings/find.cu

```diff
@@ -243,7 +243,7 @@ std::unique_ptr<column> contains_fn(strings_column_view const& strings,
   if (strings_count == 0) return make_empty_column(data_type{type_id::BOOL8});
 
   auto targets_count = targets.size();
-  CUDF_EXPECTS(targets_count > 0, "Must include at least one search target");
+  CUDF_EXPECTS(targets_count == strings_count, "strings and targets column must be the same size");
```

Comment:
```suggestion
  CUDF_EXPECTS(targets_count == strings.size(), "strings and targets column must be the same size");
```

Reply:
Is there something wrong with using the variable here?

Reply:
Not really, but the the variable is originally used in two places, but after you use `is_empty` and remove one use, it only leaves one use - and defining ...
```
strings_count
strings.size()
```
instead of just using the later seems suboptimal to me.